README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Demo
 
-[![Waallet Demo](https://i.imgur.com/78PaPdK.png)](https://www.youtube.com/watch?v=rUzqDGrcEvU)
+[![Waallet Demo](https://i.imgur.com/rOdWAJG.gif)](https://youtu.be/Ac2jmmAEG00)
```

Comment:
I think we could add a further information below this demo video. 

For example: "This showcase is for account contract controlled by passkey. To see more demonstrations, please refer to [demo page](demo.md)"